content/en/docs/Responses/AlbumID3WithSongs.md

```diff
@@ -161,11 +195,44 @@ description: >
 | `genre` | `string` | No |     | The album genre|
 | `played` | `string` | No | **Yes**    | Date the album was last played. [ISO 8601]|
 | `userRating` | `int` | No | **Yes**    | The user rating of the album. [1-5]|
+| `recordLabel` | `string` | No |  **Yes**   | The label producing the album. |
+| `musicBrainzId` | `string` | No |  **Yes**   | The album MusicBrainzID. |
+| `genres` | Array of `string` | No | **Yes**    | The list of all genres of the album. |
+| `artists` | Array of [`ArtistID3`](../artistid3) | No | **Yes**    | The list of all album artists of the album. |
+| `displayArtist` | `string` | No |  **Yes**   | The single value display artist. |
+| `releaseType` | `string` | No | **Yes**    | The type of release. If the server handle subtypes then the field should return them type and subtypes separated with " / ".|
+| `moods` | Array of `string` | No | **Yes**    | The list of all moods of the album. |
+| `sortName` | `string` | No |  **Yes**   | The album sort name. |
+| `originalReleaseDate` | `string` | No |   **Yes**   | Date the album was originally released. [ISO 8601]|
+| `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation.|
+| `discTitles` | Array of [`DiscTitle`](../disctitle) | No | **Yes**    | The list of all disc titles of the album. |
 | `song` | Array of [`Child`](../child) | No |     | The list of songs |
 
 {{< alert color="warning" title="OpenSubsonic" >}}
 New fields are added:
 
 - `played`
 - `userRating`
+- `recordLabel`
+- `musicBrainzId`
+- `genres`
+- `artists`
+- `displayArtist`
+- `releaseType`
+- `moods`
+- `sortName`
+- `originalReleaseDate`
+- `isCompilation`
+- `discTitles`
+
+**Note**: All OpenSubsonic added fields are **optionals**. But if a server support a field it **must** return it with an empty / default value when not present in it's database so that clients knows what the server supports.
```

Comment:
What do you mean by default value? Do you have examples?
If no disc title is available, empty array or array of empty strings?

Reply:
Disc title is an object ;) But yes empty array in that case or empty string for the other cases.

If there's cases where default needs explicitness please comment on it.

Reply:
If 'isCompilation' (boolean) is supported but not known for this album, what are we supposed to put?

Reply:
It would probably make sense to return false?

Reply:
Yes it's an album by default like now.